spec/requests/data_services_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe "DataServices", type: :request do
+  describe "GET /data_services" do
+    it "returns data_services" do
+      get data_services_path
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe "GET /data_services" do
```

Comment:
There's a little copy pasta here 😄 🍝   Should be something like:

```suggestion
  describe "GET /data_service" do
```

Reply:
Change now implemented 😄